install/cws-ui/logs.ftl

```diff
@@ -13,6 +13,18 @@
 	<link href="/${base}/css/dashboard.css" rel="stylesheet">
 	<script>
 
+	//STATE PERSISTANCE CONSTS
+	const username = "username"; //temporary, hardcoded value for now
+	const cwsHostVar = "CWS_DASH_LOGS_CWSHOST" + username;
+	const cwsWIDVar = "CWS_DASH_LOGS_CWSWID" + username;
+	const logLevelVar = "CWS_DASH_LOGS_LOGLVL" + username;
+	const thnVar = "CWS_DASH_LOGS_THN" + username;
```

Comment:
what is thn?  maybe better to just spell it out

Reply:
It was the ID of the "thread name" checkbox - updated